Core/LocalSettings.cs

```diff
@@ -4,6 +4,7 @@
 using System.Data;
 using System.Runtime.CompilerServices;
 using System.Text.Json;
+using RoverControlApp.Core.Settings;
```

Comment:
U can either use this using, or prefix `Settings.` in properties. Why do both?

Reply:
Done in [1c7a46c](https://github.com/wisniax/RoverControlApp/pull/16/commits/1c7a46caf7707993795a390e68fa60fa18eff9c3)